dev-infrastructure/configurations/mvp-dev-acr.bicepparam

```diff
@@ -9,21 +9,36 @@ param quayRepositoriesToCache = [
     ruleName: 'openshiftReleaseDev'
     sourceRepo: 'quay.io/openshift-release-dev/*'
     targetRepo: 'openshift-release-dev/*'
-    purgeFilter: 'quay.io/openshift-release-dev/.*:.*'
-    purgeAfter: '2d'
-    imagesToKeep: 1
     userIdentifier: 'quay-username'
     passwordIdentifier: 'quay-password'
   }
   {
     ruleName: 'csSandboxImages'
     sourceRepo: 'quay.io/app-sre/ocm-clusters-service-sandbox'
     targetRepo: 'app-sre/ocm-clusters-service-sandbox'
+    userIdentifier: 'quay-componentsync-username'
+    passwordIdentifier: 'quay-componentsync-password'
+  }
+]
+
+param purgeJobs = [
+  {
+    name: 'ocm-clusters-service-sandbox-purge'
     purgeFilter: 'quay.io/app-sre/ocm-clusters-service-sandbox:.*'
     purgeAfter: '2d'
     imagesToKeep: 1
-    userIdentifier: 'quay-componentsync-username'
-    passwordIdentifier: 'quay-componentsync-password'
+  }
+  {
+    name: 'openshift-release-dev-purge'
+    purgeFilter: 'quay.io/openshift-release-dev/.*:.*'
```

Comment:
Why do we want to purge OCP release payloads?

Reply:
I'm not really sure, @janboll do you know the answer?

Reply:
We shouldn't be.  Can we remove the purge on them?  

Unless we force customers to upgrade overlays / data plane side, we can't reap these images.

Reply:
If we're configuring pull-through cache though, it might be useful to have this as a parameter in lower environments and leverage the artifact cache to pull down the images if needed again. 

Maybe make the purge job a togglable that will default to on, but we'll disable it in lower environments?  Wdyt @stevekuznetsov

Reply:
That purge is on dev releases. these dev releases are used once for nightly build tests. So this should be fine here

Reply:
some clarification about what is in the ACR right now

* `openshift-release-dev/ocp-v4.0-art-dev` is the pull through cache from quay.io
* `openshift/release` and `openshift/release-images` is the result of oc-mirror syncing (we don't know why oc-mirror splits the images into two repos but it is documented behaviour)

`openshift-release-dev/ocp-v4.0-art-dev` exists because of the discussion to be found [here](https://redhat-external.slack.com/archives/C075PHEFZKQ/p1725551028726639). these images are not used by ARO HCP testing in DEV. the other two repos are used by ARO HCP testing in DEV.

we could start using the pull-through cache for ARO HCP DEV testing. we could even make the pullthrough cache look like the result of oc-mirror. but @janboll raised the concern that we would have no feedback on oc-mirror accuracy in DEV.

Reply:
i would not delay this PR because of this. while this needs discussion for sure, it is not a change in behaviour since @tony-schndr is just revamping the way we configured this all along.

Reply:
It might be good to capture the above data in a comment in the template (though I generally dislike comments, this feels like great context for the future)

Reply:
This isn't right. If you look at the release payload for OCP 4.17.0 AMD64, it's broken out as follows:
- Release image: quay.io/openshift-release-dev/ocp-release
- Component images: quay.io/openshift-release-dev/ocp-v4.0-art-dev

We should **not ** be pruning images in `openshift-release-dev/**` as that'll break customers.

Reply:
Oh we're rewriting paths aren't we?  So we're actually rewriting `openshift-release-dev/ocp-release` -> `openshift/release-images` and `openshift-release-dev/ocp-release` -> `openshift/release`.  

I guess that works then.

Reply:
> we're rewriting paths
oc-mirror is doing that. afaik there is no choice